citylots_bench_test.go

```diff
@@ -5,8 +5,6 @@ import (
 )
 
 func BenchmarkCityLots(b *testing.B) {
-	var localMatches []X
```

Comment:
The idea of the `localMatches` / `topMatches` is to avoid compiler optimizations.. (see https://teivah.medium.com/how-to-write-accurate-benchmarks-in-go-4266d7dd1a95 - `Not Being Careful about Compiler Optimizations`)

Reply:
Oh, OK, will put back.